website/content/en/docs/upgrading-sdk-version/v1.0.0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: v1.0.0
+weight: 999000000
+---
+
```

Comment:
Similar comment here about incorporating changes from v1.0.0-alpha.1 and v1.0.0-alpha.2 into a `Since v0.19` section. And then leave these existing headers would be duplicated in a `Since v1.0.0-alpha.2` section.

WDYT?